src/components/ImageUploader/ImageUploader.tsx

```diff
@@ -26,10 +29,19 @@ const ImageUploader: React.FC<ImageUploaderProps> = ({ onClose }) => {
         }
     };
 
-    const handleUpload = () => {
+    const handleUpload = async () => {
         if (fileList) {
-            console.log('Uploading images:', fileList);
-            alert('Images uploaded successfully!');
+            let success = false;
+            const formData = new FormData();
+            images?.forEach((image: ImageState, index: number) => formData.append(`images[${index}]`, image?.file));
+            try {
+                const { data } = await api.post('/images/predictions/', formData);
+                setFileNames(data?.images);
```

Comment:
We will need to update how we manage the response once the API is fully up and running but we can leave as is until then